OPG3 current statues

Jenssile opened the pull request:
this is what we have for now.

Cat-Lon committed:
OPG 3 start

Cat-Lon committed:
OPG 3 

Program written mostly along the manual

Cat-Lon committed:
OPG 3 Now with more logic!

moved some things around to make more sense

MadsBock commented:
Super. Godt at i har oprettet en pull request allerede nu. Tilføj mig som Reviewer når i gerne vil have feedback.

Cat-Lon committed:
OPG 3 The final solution

added comment with answer to the final question

Jenssile commented:
please have a look at it now i cant add you as a reviewer for some reason.

MadsBock reviewed (commented):
Koden er ganske fin. Jeg foreslår at i kigger på at lave således at de to parter kan sende beskeder frem og tilbage til hinanden, uafhængigt af hinanden.

Cat-Lon committed:
TEST

Chat test

Cat-Lon committed:
OPG 3 Chat

Back and forth communication added, fixed a thing where readKey() would break the first part of a message

Cat-Lon commented:
Dette burde virke

Jenssile commented:
Can you have a look at the changes. i still cant add you as a reviewer.

MadsBock commented on Opgave-3-Async/Opgave-3-Async/Program.cs:
```diff
@@ -9,8 +9,12 @@ class Program
+            {
+                Server(listener);
+                Client();
+            }
```
Hvorfor er dette i et endless loop? Det virker meget error prone. Hvis ikke det var fordi at Client() funktionen blocker med Console.ReadLine ville i blive ved med at oprette nye servere og clients uendeligt.

Cat-Lon replied:
Jeg har nu lavet en version a programmet som virker efter hvad du har beskrevet

MadsBock reviewed (commented):
Godt arbejde, men det var stadig ikke helt det jeg mente.
Lige nu kan serveren kun modtage beskeder fra clienten. Den som er server skal også kunne sende beskeder tilbage til Clienten igen.

Cat-Lon committed:
OPG 3 (2WAY) TEST

This should be a functioning 2way chat system where you only need to run either client() or server()

Cat-Lon committed:
OPG 3 (2WAY) TEST 2

work you piece of shit

Cat-Lon committed:
OP3 (2WAY) finished

working version has been made with many changes
